.github/workflows/build-windows.yml

```diff
@@ -141,16 +141,19 @@ jobs:
           # Run GTSAM_UNSTABLE tests
           cmake --build build -j4 --config ${{ matrix.build_type }} --target check.base_unstable
           # Compile. Fail with exception
-          # cmake --build build -j4 --config ${{ matrix.build_type }} --target check.geometry_unstable
+          # cmake --build build -j1 --config ${{ matrix.build_type }} --target check.geometry_unstable
           # Compile. Fail with exception
-          # cmake --build build -j4 --config ${{ matrix.build_type }} --target check.linear_unstable
+          # cmake --build build -j1 --config ${{ matrix.build_type }} --target check.linear_unstable
```

Comment:
Additionally, these test commands have not been uncommented. We would like to see all these tests passing.

Reply:
These tests are not passing, that why they on comments. On each group of test, there is some tests that not passing.

Reply:
If these tests are commented out, there's no point in making changes to them right? We can change these when we uncomment them. :-) Currently this only makes the PR hard to review :-(

Reply:
I will convert it back to -j4 because it commented